### PR TITLE
Header view login

### PR DESCRIPTION
**Pull request by pdisi**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -1,2 +1,6 @@
 @import 'bootstrap';
 
+img{
```
Comment (KGorham05) [inline app/assets/stylesheets/application.scss:3]: I would personally give this a more specific name. logo-img or header-img or something

File: app/javascript/components/components/Header.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            {!logged_in &&
+            <>
+            <a href={sign_in_route}>Login</a>
+            <a href={new_user_route}>Sign Up</a> 
+            </>
+            }
+        </header>
+    )
+}
+}
```
Comment (KGorham05) [inline app/javascript/components/components/Header.js:28]: Looks like this could use some formatting but otherwise LGTM

**Review by KGorham05 — APPROVED** [review]
Minor feedback, but LGTM!

File: app/javascript/components/App.js [hunk 3]
```diff
@@ -17,6 +18,9 @@ class App extends React{
     } = this.props
     return (
       <>
+      <Router>
+      <Header {...this.props}/>
+      </Router>
```
Comment (sjproctor) [inline app/javascript/components/App.js:23]: You should prolly remove your `h1` from App.js

**Review by sjproctor — APPROVED** [review]
Be diligent on your indentation!

**Pull request merged**